rplugin/python3/deoplete/sources/deoplete_go.py

```diff
@@ -79,13 +79,13 @@ def gather_candidates(self, context):
             for complete in result[1]:
                 _class = complete['class']
                 word = complete['name']
-                info = complete['type'].replace('func', '')
+                info = complete['type']
+                _abbr = (word + sep + info).replace(' func', '')
```

Comment:
Is this space (before the word `func`) intended? `replace(' func', '')`

(just checking)